src/modrinth/mod.rs

```diff
@@ -0,0 +1,118 @@
+pub mod converters;
+
+use crate::{
+    project::{Author, Project},
+    request::Asset,
+    version::Version,
+    Container, Platform,
+};
+use async_trait::async_trait;
+use ferinth::Ferinth;
+
+pub type ID = String;
+
+#[derive(Debug, thiserror::Error)]
+pub enum Error {
```

Comment:
ModrinthError instead of Error ?

Reply:
After splitting into crates, it will namespaced as `dotium_mr::Error` which is the way I always do errors. I find that every other crate always uses `Error` with no modifications which is why I do that too.